locales/PT_br.json

```
@@ -0,0 +1,139 @@
+{
+	"nav": {
+		"about": "Sobre",
+		"faq": "FAQ",
+		"credits": "Créditos",
+		"progress": "Progresso"
+	},
+	"hero": {
+		"subtitle": "Servidores de jogos",
+		"title": "Recreated",
+		"text": "Pretendo é uma reposição gratuíta e de código aberto para os servidores de ambos 3DS e Wii U, permitindo conectividade online para todos, mesmo após os servidores originais serem descontinuados",
+		"buttons": {
+			"readMore": "Mostrar mais"
+		}
+	},
+	"aboutUs": {
+		"title": "Sobre nós",
+		"paragraphs": [
+			"Pretendo é um projeto de código aberto que visa recriar a Nintendo Network para o 3DS e Wii U usando design de sala limpa.",
+			"Como os nossos serviços serão gratuitos e de código aberto, eles podem continuar existindo por muito mais tempo após o inevitável encerramento da Nintendo Network."
+		]
+	},
+	"progress": {
+		"title": "Progresso",
+		"paragraphs": [
+			"Atualmente estamos trabalhando no Miiverse, juntamente com nossos servidores de contas e sua integração com os serviços.",
+			"Para o 3DS também estamos trabalhando no Mario Kart 7, com o desejo de trabalhar em outros jogos quando possível."
+		]
+	},
+	"faq": {
+		"title": "Perguntas frequentes",
+		"text": "Para a fácil obtenção de informações, aqui estão algumas perguntas comuns que nos são feitas.",
+		"QAs": [
+			{
+				"question": "O que é Pretendo?",
+				"answer": "Pretendo é uma reposição de código aberto da Nintendo Network que visa fornecer servidores personalizados para a família de consoles Wii U e 3DS. Nosso objetivo é preservar a funcionalidade online desses consoles, para permitir que os jogadores continuem a experienciar seus jogos favoritos do Wii U e 3DS em sua capacidade máxima."
+			},
+			{
+				"question": "Meus NNIDs existentes funcionarão no Pretendo?",
+				"answer": "Infelizmente não. Os NNIDs existentes não funcionarão no Pretendo, pois apenas a Nintendo mantém acesso aos seus dados de usuário; embora uma migração de NNID para PNID seja teoricamente possível, seria arriscado e exigiria dados confidenciais do usuário que não desejamos manter."
+			},
+			{
+				"question": "Como eu uso o Pretendo??",
```

There's one too many question marks here
```suggestion
				"question": "Como eu uso o Pretendo?",
```